### ISSUE TITLE
onlyOnRoot not redirecting based on cookie in static target

### ISSUE THREAD

**Issue by divine** [nuxt-modules/i18n#887]
### Version
nuxt-i18n: 6.15.0
nuxt: 2.14.4

#### [mode](https://nuxtjs.org/guides/configuration-glossary/configuration-mode): 
 - [x] universal (static)

### Nuxt-i18n configuration
```
i18n: {
    strategy: 'prefix',
    detectBrowserLanguage: {
      useCookie: true,
      alwaysRedirect: true,
      onlyOnRoot: true
    },
}
```

### Reproduction Link

Demo: http://silent-watch.surge.sh/ru
Github: https://github.com/divine/i18n-static-redirect

It works on CodeSandbox because they don't support static target and runs in dev mode.

### Steps to reproduce

When locale is changed cookie is being changed.

However on root path with `onlyOnRoot` option doesn't redirect based on stored cookie.

### What is Expected?

When visiting https://silent-watch.surge.sh/ru going back to https://successful-direction.surge.sh should redirect to `ru`.

### What is actually happening?

Redirection to /en

**Comment by rchl** [comment]
Right, so it should redirect before updating the cookie in this case.

**Comment by williamweckl** [comment]
Exact same issue here. After I've updated to 6.15.0, to make it work with onlyOnRoot I had to delete all the cookies. After deleting all cookies it worked as expected.

**Comment by rchl** [comment]
It's the #805 fix that doesn't play well with the latest changes.

The #805 is creating a redirect route from `/` to the default locale. I think I can just change that from a **redirect route** to a normal route. It should still trigger redirect but wouldn't force it to redirect to specific (default) locale.

**Comment by rchl** [comment]
Fix released in [v6.15.1](https://github.com/nuxt-community/i18n-module/releases/tag/v6.15.1)
